app/views/game/mini.scala

```diff
@@ -85,8 +85,8 @@ object mini {
   def renderState(pov: Pov) =
     pov.game.variant match {
       case Variant.Chess(_) | Variant.FairySF(_) | Variant.Samurai(_) | Variant.Togyzkumalak(_) |
-          Variant.Go(_) | Variant.Backgammon(_) =>
-        dataState := s"${Forsyth.>>(pov.game.variant.gameLogic, pov.game.stratGame)}|${orientation(pov)}|${~pov.game.lastActionKeys}"
+          Variant.Go(_) | Variant.Backgammon(_) | Variant.Abalone(_) =>
+        dataState := s"${Forsyth.boardAndPlayer(pov.game.variant.gameLogic, pov.game.situation)}|${orientation(pov)}|${~pov.game.lastActionKeys}"
```

Comment:
I think this could be an incorrectly resolved merge conflict? Check with James

Reply:
Agreed, we have changed this to .>> to get the whole fen not just board and player.